### DISCUSSION TITLE
GitHub เป็นแพลตฟอร์มที่สมบูรณ์มาก

### DISCUSSION THREAD

**Discussion by Dang2522** [community/community#140407] [Accessibility]
แหล่งรวมนักพัฒนาซอฟต์แวร์

**Comment by queenofcorgis** [comment]
Hi @Dang2522 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

_Please note for future posts: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
